lib/presto-client/index.js

```diff
@@ -1,3 +1,5 @@
+var parseUrl = require('parse-url');
```

Comment:
Are there any feature which isn't supported by standard `url` module of node.js? https://nodejs.org/api/url.html
If so, could you add comments about missing features?

Reply:
Otherwise, switch to use `url` to avoid adding dependencies.

Reply:
Thank you for feedback. I will look into this and add some additional commits when ready. I will also follow the manner of commits that you have recommended in your other comment.